composer.json

```diff
@@ -124,7 +67,8 @@
         "mikey179/vfsStream": "^1.6",
         "phake/phake": "@stable",
         "phpunit/phpunit": "~5.7",
-        "symfony/phpunit-bridge": "^3.0"
+        "symfony/phpunit-bridge": "^3.0",
+        "prestashop/composer-script-handler": "~1.0"
```

Comment:
Can you please double check, I haven't found `composer-script-handler` in the file composer.lock.